modules/zip/src/parse-zip/zip-compozition.ts

```diff
@@ -111,3 +116,74 @@ export async function addOneFile(zipUrl: string, fileToAdd: ArrayBuffer, fileNam
     )
   );
 }
+
+/**
+ * creates zip archive with no compression
+ * @param inputPath path where files for the achive are stored
+ * @param outputPath path where zip archive will be placed
+ */
+export async function createZip(inputPath: string, outputPath: string) {
```

Comment:
- Can we generalize this a bit so we don't have to make the code node specific?
```suggestion
export async function createZip(fs: FileSystem, inputPath: string, resFile: WritableFile) {
```

Reply:
As I mentioned earlier, wile there's no way to define if it's directory or not, I propose using `NodeFileSystem` there